README.md

```diff
@@ -1 +1,103 @@
-# OOP-school-library
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Thank you @HAMMAS-SALEEM for the recommendation. Will work on it.